molgenis-settings-ui/src/main/frontend/src/store/state.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+export const INITIAL_STATE = window.__INITIAL_STATE__ || {}
+
+const state = {
+  error: undefined,
+  token: undefined,
+  formFields: [],
```

Comment:
could in future use flow typing from molgenis-ui-form, we should export the types i guess